<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>361</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>301</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab1">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>RGB</string>
     </attribute>
     <widget class="QLabel" name="label_histo1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_histo2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_histo3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab2">
     <attribute name="title">
      <string>YUV</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Ã‰dition</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFiltres">
    <property name="title">
     <string>Filtres</string>
    </property>
    <addaction name="actionNoir_et_blanc"/>
    <addaction name="actionInversion"/>
    <addaction name="actionFlou"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuFiltres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionNoir_et_blanc">
   <property name="text">
    <string>Noir et blanc</string>
   </property>
  </action>
  <action name="actionInversion">
   <property name="text">
    <string>Inversion</string>
   </property>
  </action>
  <action name="actionFlou">
   <property name="text">
    <string>Flou</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
